logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/MessageRepository.kt

```diff
@@ -303,17 +305,10 @@ internal class MessageDataSource internal constructor (
         messageSizePerConversation: Int
     ): Either<CoreFailure, Flow<List<LocalNotification>>> = wrapStorageRequest {
         messageDAO.getNotificationMessage().mapLatest { notificationEntities ->
-            notificationEntities.groupBy { it.conversationId }
-                .map { (conversationId, messages) ->
-                    LocalNotification.Conversation(
-                        // todo: needs some clean up!
-                        id = conversationId.toModel(),
-                        conversationName = messages.first().conversationName ?: "",
-                        messages = messages.take(messageSizePerConversation)
-                            .mapNotNull { message -> messageMapper.fromMessageToLocalNotificationMessage(message) },
-                        isOneToOneConversation = messages.first().conversationType == ConversationEntity.Type.ONE_ON_ONE
-                    )
-                }
+            notificationMapper.fromEntitiesToLocalNotifications(
```

Comment:
Suggestion: maybe we can have a test for this func, since it's changing (I had a look, and it seems we never had one for this)